### DISCUSSION TITLE
AOSP 12.1 v415

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#2393] [General]
- July security patch

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v415'>AOSP 12.1 v415</a>.</em>

**Comment by Taogezuibang** [comment]
大神，我的Redmi 10X Pro 开启Volte后来电手机没有任何反应，屏幕指纹识别不灵敏

**Comment by summer-son** [comment]
Why is this tagged as pre-release? Is it less stable than v414?

**Comment by FerFemNemBem** [comment]
Hi,
"PixelExperience_arm64-ab-12.1-20220719-UNOFFICIAL.img" (v415 for Redmi Note 10 5G) always ends in recovery on booting. Flashing "PixelExperience_arm64-ab-12.1-20220612-UNOFFICIAL.img" (v414) the same way (fastboot, whipe data etc.) - everything is fine.
Are there any known issue with v415?
FFNB

**Reply by CGMukonde** [reply]
Same here not working takes me to recovery at boot

**Reply by AnshhSingh** [reply]
> this is problem on MTK Device

Any hopes that this will fixed in newer releases? I am Facing the same problem with my mkt device

**Reply by jackie099** [reply]
Wipe data with recovery and it'll boot again

**Reply by DavidPinilo** [reply]
Wiping the phone does nothing

**Reply by FerFemNemBem** [reply]
Same here. Stays in boot loop also after wiping all data.

**Reply by DavidPinilo** [reply]
I just flashed back the previous Pixel Experience release and I guess I'll stay there until there's a release that's confirmed to work on MTK devices.

**Comment by mariasanchez84** [comment]
Works great, thanks !

**Comment by CGMukonde** [comment]
Pixel experience v415 doesn't start at boot

**Comment by devrieshh** [comment]
thanks to make the square menu button on the bottom working on the Xiaomi redmi 8a  and Cubot king kong mini 2  in system-squeak-arm32_binder64-ab-vndklite-gogapps-secure.img
![](mtp://Xiaomi_SDM439-QRD__SN%3AEE2FA835_217329340707/SD%20card/DCIM/Camera/IMG_2022-08-23-19-19-48-753.jpg)

**Comment by Taogezuibang** [comment]
Hi， Redmi 10X Pro（bomb） Offscreen Fingerprint and Volte not work

**Comment by ghost** [comment]
Why is webview version too outdated (v95) ? 🫤

**Comment by devrieshh** [comment]
security update is september 1 ,2021 how to get the newest securety update?

**Reply by ghost** [reply]
Actually it's july 2022. Settings UI shows wrong dates due to unknown weird reasons...¯\\\_(ツ)_/¯

**Reply by devrieshh** [reply]
ok,  thanks for the replay

**Reply by iseppe** [reply]
I think the reason is to pass SafetyNet

**Reply by devrieshh** [reply]
make sense

**Comment by Docence** [comment]
Hi！I want to flash Android 12 gsi to my HTC u11+, but there is only one A partition on the device and does not support sar(system-as-root).So is there any way to boot gsi or customize  ROM? thanks!

**Reply by ghost** [reply]
https://forum.xda-developers.com/f/treble-enabled-device-development-a-only-roms.8361/

**Comment by Jafery20** [comment]
Good morning, I'm using "PixelExperience_arm64-ab-12.1-20220719-UNOFFICIAL.img" on a Samsung Galaxy A50, and the problem I have is that when I turn off the phone, or it turns off due to lack of battery, when I connect it, the boot animation appears, instead of a battery charging animation.  And when the boot animation appears, it doesn't turn on the device, it just stays in bootloop, until I manually press the volume - and power buttons to make it reboot, and turn on again.

**Comment by 0x0tyy** [comment]
Hey I'm new to GSI and ROMs in general.
Assuming no bugs, is it general consensus that phh's GSI has the best battery out of all the GSI ROMS out there? or am I mistaken?

**Comment by sadiqsalau** [comment]
I'm using honor 7x, I have an issue with dual sim 4g lte. LTE works only on sim 1. I'm able to use lte on sim 2 on android 11, 10 and 9

**Reply by AnshhSingh** [reply]
Did it work for you on the stock rom

**Comment by sadiqsalau** [comment]
Yes, every phh based gsi prior to Android 12 worked. It stays on 2g
whenever I switch the data Sim to sim2. I had to swap the Sim inside the
Sim tray to get 4g lte. 4g only works on Sim1. Thanks

**Comment by sadiqsalau** [comment]
I can't say, I'm using Huawei BND-L21 device. I only have this issue with
android 12 gsi

**Comment by Daewatar** [comment]
I'm using the RQ3A.211001.001
PixelExperience_Plus_treble_arm64_bvN-11.0-20211011-0811-UNOFFICIAL ROM on my Redmi note 10 5G. I want to upgrade it to latest Android 13 GSI. My question is How can I upgrade it. I often get notifications that says Update to Android 13 and Tap to download and install. I tap it and nothing happened. I also don't wanna wipe user data therefore I'm finding method like OTA upgrade. But I couldn't find it anywhere. So please tell me is there a method like OTA and if not how can I reserve my user data ? Can I backup and restore it like normal roms.

**Comment by AlexPashenko** [comment]
Greetings. I have a problem. On any ROM, on your AOSP, and on other ROMs based on AOSP, when I add a fingerprint, there is only a vibration, but the fingerprint is not saved. I have log. My phone Xiaomi 12pro.
Log https://drive.google.com/file/d/1IXucSvRQJtUuXnEYv-sSNrmuiowt_1wh/view?usp=drivesdk

**Comment by Cyterafle** [comment]
Hi,
On Android 12 GSIs with Huawei P20 Lite I have Bluetooth issues : It works but I can't pair Bluetooth devices

**Comment by randomguyfromAustria** [comment]
When does v416 come out?

**Comment by phhusson** [comment]
Not

15 sept. 2022 18:23:46 randomguyfromAustria ***@***.***>:

**Reply by nicman23** [reply]
hey sorry do you mean that there wont be any more support?

**Reply by DerGoogler** [reply]
I think, this is an yes

**Comment by Roniere001** [comment]
It was very good on My Galaxy SM-T510 but the version with the gaaps showed a failure in the configuration APP.

**Comment by RealACJoshua** [comment]
My Tecno Lb8a's mobile data wouldn't work after I flashed this version

**Comment by Davi89rony** [comment]
> * patch de segurança de julho
> 
> _Esta discussão foi criada a partir da versão [AOSP 12.1 v415](https://github.com/phhusson/treble_experimentations/releases/tag/v415) ._


Hello, help me, I have twrp, I put Gsis, I don't know, take it, help me

**Comment by Davi89rony** [comment]
Hello, help me, I have twrp, I put Gsis, I don't know, take it, help me
